docs/tutorials/tacho-motors.md

```diff
@@ -7,13 +7,17 @@ group: hardware-standard
 {:toc}
 
 {% include icon.html type="danger" %}
-This tutorial has been updated to kernel release 10. This is different from the
-version that comes pre-installed on the current image files. Check your kernel
-version by running `uname -r`. If it does not contain `-10-ev3dev-` in the version,
-upgrade your kernel and try again before filing a bug.
+This tutorial has been updated for changes introduced in ev3dev kernel release
+10. **If you are running on an image file from before 2016-10-17 and have
+not manually upgraded your kernel, you will need to upgrade before continuing.**
+You can check your kernel version by running `uname -r`. If it does not contain
+`-10-ev3dev-` or higher, you are not running a supported version.
 {: .alert .alert-danger}
 
-This tutorial uses a command line shell to demonstrate how to use the [tacho-motor] class.
+This tutorial uses a command line shell to demonstrate how to use the
+[tacho-motor] class. Although there are many devices that can be used as tacho
+motors, the most common ones are the standard "EV3 Large Servo Motor" and
```

Comment:
Are you sure you want to say "Servo" here? I know that is the official LEGO name for the motors, but it makes people think they should be using the `servo-motor` class instead of the `tacho-motor` class.